src/hdtAssetValidator.cpp

```
@@ -0,0 +1,989 @@
+			}
+
+			bodyStream << "  " << report.xmlImprovedCount << " improved file(s) written.\n";
+		}
+
+		// Stop timer
+		auto wallEnd = std::chrono::steady_clock::now();
+		double elapsedSec = std::chrono::duration<double>(wallEnd - wallStart).count();
+		report.elapsedSeconds = elapsedSec;
+
+		// Errors and warnings index (appended after body for quick reference)
+		std::ostringstream tailStream;
```

Clang (14.0.6)

[warning] variable 'tailStream' is not initialized

(cppcoreguidelines-init-variables)